src/evict/evict_lru.c

```diff
@@ -891,17 +880,63 @@ __evict_clear_walk(WT_SESSION_IMPL *session)
      */
     btree->evict_ref = NULL;
 
+    if (clear_pos) {
+        btree->evict_pos = WT_NPOS_INVALID;
```

Comment:
Add call to __wt_btree_clear_evict_npos here.

Reply:
done